common/djangoapps/django_comment_common/utils.py

```diff
@@ -1,9 +1,17 @@
 """
 Common comment client utility functions.
 """
-
 from django_comment_common.models import Role, FORUM_ROLE_ADMINISTRATOR, FORUM_ROLE_MODERATOR, \
     FORUM_ROLE_COMMUNITY_TA, FORUM_ROLE_STUDENT
+from request_cache.middleware import request_cached
+
+from .models import CourseDiscussionSettings
+from openedx.core.djangoapps.course_groups.cohorts import get_legacy_discussion_settings
+
+
+NOT_DIVIDED = CourseDiscussionSettings.NONE
```

Comment:
Nit: I personally find this to be less clear and redundant.  It seems like we are just renaming the constants and you still need to import CourseDiscussionSettings.  I believe that these are defined in the right place initially and that moving them to utils is out of context.  The context shift and renaming could cause confusion.

I am not so concerned to block the PR for this though.